templates/prometheus.yml.j2

```diff
@@ -22,16 +22,20 @@ scrape_configs:
     static_configs:
       - targets: ['speedtest:9798']
 
+{% if shelly_plug_enable %}      
   - job_name: 'shelly-plug'
     metrics_path: /metrics
     scrape_interval: 1m
     static_configs:
       - targets: ['172.17.0.1:9924']
+{% endif %}
 
+{% if starlink_enable %}      
```

Comment:
Looks like there are some trailing spaces on this line (and a few others). Can you clean that up and then this can be merged?